core/app/assets/js/organisms/error-boundary.js

```diff
@@ -0,0 +1,58 @@
+import Dialog from 'elementor-app/ui/dialog/dialog';
+
+// In the current time there is no solution to use "getDerivedStateFromError" static method with functional component
+// That is why this component is a class component.
+// @link https://reactjs.org/docs/hooks-faq.html#do-hooks-cover-all-use-cases-for-classes
+export default class ErrorBoundary extends React.Component {
+	static propTypes = {
+		children: PropTypes.any,
+		title: PropTypes.string,
+		text: PropTypes.string,
+		learnMoreUrl: PropTypes.string,
+	};
+
+	static defaultProps = {
+		title: __( 'App could not be loaded', 'elementor' ),
+		text: __( 'We’re sorry, but something went wrong. Click on ‘Learn more’ and follow each of the steps to quickly solve it.', 'elementor' ),
+		learnMoreUrl: 'https://go.elementor.com/app-general-load-issue/',
```

Comment:
It's better that these values won't be provided as default values, but will be given from the implementation

Reply:
It must have default values because it's a default generic error.